app/controllers/application_questions_controller.rb

```diff
@@ -0,0 +1,74 @@
+class ApplicationQuestionsController < ApplicationController
+  before_action :set_application_question, only: [:show, :edit, :update, :destroy]
+
+  # GET /application_questions
+  def index
+      @application_questions = ApplicationQuestion.all
+  end
+
+  # GET /application_questions/1
+  def show
+  end
+
+  # GET /application_questions/new
+  def new
+    @application_question = ApplicationQuestion.new
+    render :edit
+  end
+
+  # GET /application_questions/1/edit
+  def edit
+    render :edit
+  end
+
+  # POST /application_questions
+  def create
+    params[:application_question][:created_at] = Time.now
+    params[:application_question][:updated_at] = Time.now
+    @application_question = ApplicationQuestion.new(application_question_params)
+
+    respond_to do |format|
```

Comment:
create and update are almost identical - see if you can either merge them or at least extract the common stuff into another method - DRY (https://en.wikipedia.org/wiki/Don%27t_repeat_yourself#:~:text=Don't%20repeat%20yourself%20(DRY,data%20normalization%20to%20avoid%20redundancy.)